lib/puppet/provider/openldap_database/olc.rb

```diff
@@ -37,6 +38,11 @@ def self.instances
           rootpw = Base64.decode64(line.split(' ')[1])
         when /^olcSuffix: /
           suffix = line.split(' ')[1]
+        when /^olcLimits: /
+          Puppet.debug "LIMIT: #{line}"
+          foo, limit = line.match(/^olcLimits:\s+(\{\d+\})?(.+)$/).captures
```

Comment:
Why do you capture `foo` if you don't use it?
I think this would do the same:
```ruby
limit = line.match(/^olcLimits:\s+\{\d+\}?(.+)$/).captures[0]
```